apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -280,6 +280,7 @@ export const SignedVerdictOverview: React.FC = () => {
       modification = formatMessage(
         m.sections.modifyDatesModal.validToDateAndIsolationToDateAreTheSame,
         {
```

Comment:
I create a refactoring task to move these modify dates modals out of the signed verdict overview. https://app.asana.com/0/1199153462262248/1202203687881448/f